docs/manage/requesting_data_as_a_collaborator.md

```diff
@@ -1,63 +1,14 @@
-
-This document provides the procedure to request access to such kinds of data. Most of the procedure is common; differences between them are labelled **linked** and **de-linked**.
-
-## Setup GPG ##
-
-We will send and receive data encrypted/signed using GPG.
-1. The steps for creating a GPG keypair are at https://www.gnupg.org/gph/en/manual/c14.html.
-1. Create a keypair and export it.
-1. Send me (@shankari, shankari@eecs.berkeley.edu) the public key via email.
-
-## Data request ##
-
-### De-linked ###
-Next, you need to formally request access by filling out a pdf form.
-
-1. I will send you an encrypted version of the form you need to fill out and a copy of *my* public key.
-1. Decrypt it using https://www.gnupg.org/gph/en/manual/x110.html.
-1. Fill it out and sign it physically.
-1. Also sign it electronically https://www.gnupg.org/gph/en/manual/x135.html
-1. Encrypt it using my public key https://www.gnupg.org/gph/en/manual/x110.html and send it to me
-
-If all of this works, we know that we have bi-directional encrypted communication over email. Make sure to encrypt any privacy sensitive information (e.g. subsets of data for debugging) that you send to me in the future.
-
-### Linked ###
-You need to send me a copy of your IRB approval and your consent document to ensure that you have permission to collect data.
-
-## Data retrieval ##
-
-### De-linked ###
-1. As you can see from the consent document, you can get access to data that is time-delayed by 1 months.
-1. I will upload an encrypted zip file with ~ 3 months of data to google drive and send you a link.
-
-Note that this data is very privacy-sensitive, so think through the answers carefully on the request form carefully and make sure that you follow them. Treat the data as you would like your data to be treated.
-
-### Linked ###
-1. I will upload an encrypted zip file with all your data to google drive and send you a link.
-
-
-### Both ###
-1. You need to decrypt it just like you decrypted the pdf form https://www.gnupg.org/gph/en/manual/x110.html.
-1. When unzipped, the data consists of multiple json files, one per user.
-1. The data will typically contain both raw sensed data (e.g. `background/location`) and processed data (e.g. `analysis/cleaned_trip`)
-1. Data formats for the json objects are at `emission/core/wrapper` (e.g. `emission/core/wrapper/location.py` and `emission/core/wrapper/cleanedtrip.py`)
-
-## Data analysis ##
+## Data Analysis - Server ##
```

Comment:
This is actually the deprecated method now. We sometimes internally use the user specific dumps to reproduce errors, but for external users, they either get the mongodump, or download csv files from their admin dashboard.

Reply:
Made a change to specify this is for internal testing only!  Let me know if I should be more specific about it being a deprecated method.

Should I add a footnote about working with CSV's?  I've only worked with the `mongodump` format, but could ask around for helping writing a section on that process.